components/Footer.js

```diff
@@ -6,25 +6,25 @@ export default function Footer() {
 
   const nav = (
     <Nav>
-      <a external target="_blank" rel="noreferrer" href="https://app.clarity.so/%E2%82%A1ABIN">
```

Comment:
was seeing

`Warning: Received `true` for a non-boolean attribute `external`.`

locally, so I removed these attributes. Don't think they are valid HTML